diff --git a/node_modules/ember-source/dist/packages/@ember/-internals/metal/index.js b/node_modules/ember-source/dist/packages/@ember/-internals/metal/index.js
index 8b9ac25..21fb17f 100644
--- a/node_modules/ember-source/dist/packages/@ember/-internals/metal/index.js
+++ b/node_modules/ember-source/dist/packages/@ember/-internals/metal/index.js
@@ -299,6 +299,8 @@ function deactivateObserver(target, eventName, sync = false) {
 
   if (activeObservers !== undefined) {
     let observer = activeObservers.get(eventName);
+    if (!observer)
+      return;
     observer.count--;
 
     if (observer.count === 0) {
